src/api/pastel-rpc/wallet.ts

```diff
@@ -306,4 +309,39 @@ export class WalletRPC {
       }
     }
   }
+
+  async importPrivKey(key: string, rescan: boolean): Promise<string> {
+    if (key.startsWith('p-secret-extended-key')) {
+      try {
+        const { result } = await rpc<TZPrivKeyResponse>('z_importkey', [
+          key,
+          rescan ? 'yes' : 'no',
+        ])
+        return result.address
+      } catch (err) {
+        return ''
```

Comment:
Updated

Reply:
By don't hide error I mean don't return empty stings, arrays when error happens.
When someone is going to use your function he can't know if it can throw error, or if it may return empty string as a sign or error, so he has to read implementation.
And if we agree to throw errors when error happens I think it will be easier, it's a common way to do.

Reply:
Ok, I have updated to throw errors